src/views/AutogenView.vue

```diff
@@ -143,7 +143,6 @@
           <a-form
             :form="form"
             @submit="handleSubmit"
-            v-show="dataView || !currentAction.groupAction || this.selectedRowKeys.length === 0"
```

Comment:
@davidjumani can you check if we can put focus on a-inpur for the auto-generated form?

Reply:
Done!